InterfacePython/Tasmanian.py

```diff
@@ -41,6 +41,7 @@
 SparseGrid = TasmanianSparseGrid
 
 import TasmanianDREAM as DREAM
+import TasmanianOPT as OPT
```

Comment:
Naming conventions:

TasmanianSG.py and TasmanianOPT.py are fine

But we  should have:
```
import TasmanianOPT as Optimization
```
which is more expressive.

Reply:
Changed.